src/tennisSimulator/Player.java

```diff
@@ -0,0 +1,62 @@
+package tennisSimulator;
+
+public class Player implements Comparable<Player> {
+
+    public static final String MEN_ORGANIZATION = "ATP";
+    public static final String WOMEN_ORGANIZATION = "WTP";
+    public static final String MEN = "men";
```

Comment:
I think you want to have singular form here - "man".
Because gender usually contains singular form: man, woman